server/src/main/java/org/apache/druid/server/coordinator/loading/CuratorLoadQueuePeon.java

```diff
@@ -230,12 +230,9 @@ public void run()
         final String path = ZKPaths.makePath(basePath, segmentHolder.getSegmentIdentifier());
         final byte[] payload = jsonMapper.writeValueAsBytes(segmentHolder.getChangeRequest());
         curator.create().withMode(CreateMode.EPHEMERAL).forPath(path, payload);
-        log.debug(
-            "ZKNode created for server to [%s] %s [%s]",
-            basePath,
-            segmentHolder.getAction(),
-            segmentHolder.getSegmentIdentifier()
-        );
+        if (log.isDebugEnabled()) {
+          log.debug("ZKNode created for server to [%s] %s [%s]", basePath, segmentHolder.getAction(), segmentHolder.getSegmentIdentifier());
```

Comment:
Nit: Line is too long.
```suggestion
          log.debug(
              "ZKNode created for server to [%s] %s [%s]",
              basePath, segmentHolder.getAction(), segmentHolder.getSegmentIdentifier()
          );
```